src/main/scala/sbtprotoc/ProtocPlugin.scala

```diff
@@ -271,7 +269,8 @@ object ProtocPlugin extends AutoPlugin {
         .value,
       PB.protocExecutable := {
         import CacheImplicits._
-        import sbt.librarymanagement.LibraryManagementCodec._
+        import sbt.librarymanagement.LibraryManagementCodec.ModuleIDFormat
+        implicit val fileStringIso = IsoString.iso[File](_.toString, file)
```

Comment:
mimics the sbt 1.3 behavior brought in via https://github.com/sbt/sbt/blob/8c160c31859347888d1c5bb1f2c71e269d66245d/sbt/src/main/scala/package.scala#L48: https://github.com/sbt/sbt/blob/5afe8821d87dca9ad878401f40606a8003e65656/main/src/main/scala/sbt/nio/FileStamp.scala#L130-L158, as I couldn't find any `JsonFormat[File]` anywhere (probably because there is no canonical form).